index.html

```diff
@@ -0,0 +1,23 @@
+
+
+<h1 id = "title"> 
+    PEDRO JOAQUIN CHAMORRO CARDENAL
+    </h1>
+    <br>
+
+<Main id = "main">
+<link rel="stylesheet" href="styles.css">
+    <div id= "img-div" div> 
+      <img id = "image"> <img src ="https://www.confidencial.digital/wp-content/uploads/2022/01/PEDRO-JOAQUIN-CHAMORRO1.jpg.webp"> <figcaption id ="img-caption" >Last picture taken to Pedro Joaquin Chamorro </figcaption>
+<head> 
+  <div id="tribute-info">
+  <ul>
+ <li>   Pedro Joaquin Chamorro exemplified the values and principles of full democracy and free elections: political pluralism and accountability; no successive terms; separation of public from private interests; the struggle against corruption; reforms with social justice; freedom of the press.</li>
+  
```

Comment:
Please have in mind that a <li> tag is used to have several items in the list. Here we just have one item in the list.

Reply:
- [ ] Please give more explanation and tell the student `why` you give these changes to him/her then the student will understand easily :+1: